spec/meilisearch/client/multi_search_spec.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+RSpec.describe 'MeiliSearch::Client - Multiple Index Search' do
+  before do
+    client.create_index('books')
+    client.create_index('movies')
+  end
+
+  it 'does a custom search with two different indexes' do
+    response = client.multi_search([
+                                     { index_uid: 'books', q: 'prince' },
+                                     { index_uid: 'movies', q: 'prince' }
+                                   ])
```

Comment:
Should we not add `{ queries: [...] }` since we might have other parameters in the future?